packages/protocol/test/team/TimelockTokenPool.t.sol

```diff
@@ -72,62 +94,82 @@ contract TestTimelockTokenPool is TaikoTest {
 
         pool.grant(
             Alice,
-            TimelockTokenPool.Grant(10_000e18, 0, 0, 0, unlockStart, unlockCliff, unlockPeriod)
+            TimelockTokenPool.Grant(
+                10_000e18, strikePrice1, 0, 0, 0, unlockStart, unlockCliff, unlockPeriod
+            )
         );
         vm.prank(Vault);
         tko.approve(address(pool), 10_000e18);
+        vm.prank(Alice);
+        usdc.approve(address(pool), 10_000e18 / ONE_TKO_UNIT * strikePrice1);
 
         (
             uint128 amountOwned,
             uint128 amountUnlocked,
             uint128 amountWithdrawn,
-            uint128 amountWithdrawable
+            uint128 amountToWithdraw,
+            uint128 costToWithdraw
         ) = pool.getMyGrantSummary(Alice);
         assertEq(amountOwned, 10_000e18);
         assertEq(amountUnlocked, 0);
         assertEq(amountWithdrawn, 0);
-        assertEq(amountWithdrawable, 0);
+        assertEq(amountToWithdraw, 0);
+        assertEq(costToWithdraw, 0);
 
         vm.warp(unlockCliff);
 
-        (amountOwned, amountUnlocked, amountWithdrawn, amountWithdrawable) =
+        (amountOwned, amountUnlocked, amountWithdrawn, amountToWithdraw, costToWithdraw) =
             pool.getMyGrantSummary(Alice);
         assertEq(amountOwned, 10_000e18);
         assertEq(amountUnlocked, 0);
         assertEq(amountWithdrawn, 0);
-        assertEq(amountWithdrawable, 0);
+        assertEq(amountToWithdraw, 0);
+        assertEq(costToWithdraw, 0);
 
         vm.warp(unlockCliff + 1);
 
-        (amountOwned, amountUnlocked, amountWithdrawn, amountWithdrawable) =
+        (amountOwned, amountUnlocked, amountWithdrawn, amountToWithdraw, costToWithdraw) =
             pool.getMyGrantSummary(Alice);
+
+        // TODO(dani): who did you figured out this number? It's really hard to maintain
```

Comment:
It was you who figured this out, not me !
https://github.com/taikoxyz/taiko-mono/pull/15006/commits/73f27713052f43b38aea72d874ae5e0cbe87d0b4

And I agree, it is really ahrd to maintain.